app/services/statistic_service.rb

```diff
@@ -5,6 +5,7 @@ def self.reporting(recruitment_cycle:)
       courses: CourseReportingService.call(courses_scope: recruitment_cycle.courses),
       publish: PublishReportingService.call(recruitment_cycle_scope: recruitment_cycle),
       allocations: AllocationReportingService.call(recruitment_cycle_scope: recruitment_cycle),
+      rollover: (RolloverReportingService.call if RecruitmentCycle.next_recruitment_cycle.present?),
```

Comment:
We should make the call here regardless of whether the next cycle is present or not and have the service decide what to return in that case. It should probably return an empty object `{}` rather than nil.

Reply:
Thanks Graham!